src/routes/new.jsx

```diff
@@ -389,7 +398,13 @@ export default function NewIssuePage() {
 
   return (
     <>
-      <Nav avatarUrl={authState.user?.avatar_url} onLogout={logout} />
+      <Nav
+        avatarUrl={authState.user?.avatar_url}
+        onLogout={() => {
+          logout();
+          navigate("/login");
```

Comment:
Instead of relying on state update for rendering the `<Login>`, we redirect to `/login`